src/planner/expand_hypertable.c

```diff
@@ -385,25 +385,59 @@ constify_timestamptz_op_interval(PlannerInfo *root, OpExpr *constraint)
  * Expressions with value on the left side will be switched around
  * when building the expression for RestrictInfo.
  *
- * Caller must ensure that only 2 argument time_bucket versions
```

Comment:
This comment is still relevant, right? Only 2 argument time_bucket versions are acceptable

Reply:
Not really, the caller doesn't have to ensure this anymore. I moved these check inside this function itself.